lametro/wagtail_hooks.py

```diff
@@ -23,22 +38,189 @@ class AlertAdmin(ModelAdmin):
     )
 
 
+BOOLEAN_CHOICES = (
+    ("true", "Yes"),
+    ("false", "No"),
+)
+
+
+class NoAddModelPermissionPolicy(ModelPermissionPolicy):
+    """
+    Model permission that doesn't allow creating new instances.
+    """
+
+    def user_has_permission(self, user, action):
+        if action in ("add", "delete"):
+            return False
+        return user.has_perm(self._get_permission_name(action))
+
+
+class BoardMemberFilterSet(django_filters.FilterSet):
```

Comment:
This is a `django-filter` filter set, which I'm using to add custom filters for whether a person is a current board member and what body they belong to, on the board member detail snippet listing page.